app/database/models/user.py

```diff
@@ -106,11 +118,20 @@ def json(self):
             "photo_url": self.photo_url,
             "need_mentoring": self.need_mentoring,
             "available_to_mentor": self.available_to_mentor,
+            "is_organization_rep": self.is_organization_rep,
+            "additional_info": self.additional_info,
+            "timezone": self.timezone,
         }
 
     def __repr__(self):
-        """Returns the user's name and username. """
-        return f"User name {self.name} . Username is {self.username} ."
+        """Returns the user's name, username, whether or not they are an organization representative and their timezone. """
```

Comment:
Requires approval from @isabelcosta

Reply:
Yep..., I actually think it'll be too much of a problem if we don't include the `is_organization_rep` or `Timezone` in here since they can be abstracted later once we grab the user data using `GET /user` api call 🤔. I will just remove them for now 😁.